site/docs/v1/tech/themes/_theme-troubleshooting.adoc

```diff
@@ -1,3 +1,23 @@
-If you happen to get into a situation where you have edited a template and it is causing errors that are preventing you from logging in, you can override the use of the UI templates to render a login form that lets you log in. 
 
-To do this, open your browser and access your FusionAuth admin UI. This will redirect you to the broken `/oauth2/authorize` page. Click in your browsers address bar and scroll to the end. Finally, add the String `&bypassTheme=true` to the end of the URL and hit the Enter key. This should render the default login page that ships with FusionAuth and allow you to log in and fix any errors you have.
+// This expects to be brought in under a == heading, otherwise jekyll will complain.
+
+=== Theme Errors Prevent Login
+
+If you happen to get into a situation where you have edited a template and it is causing errors that are preventing you from logging in, you can override the use of the UI templates to render a login form that lets you log in. To do this: 
```

Comment:
This reads a bit clunky... maybe
In the situation where you have broken your themed template, especially the ones that allow login to occur, fear not - you can bypass your faulty template.  To do this: